Velar/Velar/Classes/VelarPresenter/VelarPresenter.swift

```diff
@@ -97,6 +99,7 @@ public final class VelarPresenter {
         baseView.viewPresenter.present(view: view, animate: animate)
         
         alphaAnimator.transitionAlpha(show: true, view: backgroundOverlayView, animated: animate, completion: nil)
+        delegate?.didPresent()
```

Comment:
Should this be in the completion block like the hide function or should it be named willPresent?

Reply:
completion block -- will update